addons/website_slides/models/slide_slide.py

```diff
@@ -159,6 +169,7 @@ class Slide(models.Model):
     url = fields.Char('Document URL', help="Youtube or Google Document URL")
     document_id = fields.Char('Document ID', help="Youtube or Google Document ID")
     link_ids = fields.One2many('slide.slide.link', 'slide_id', string="External URL for this slide")
+    slide_resource_ids = fields.One2many('slide.slide.resource', 'slide_id', string="Additional Resource for this slide")
```

Comment:
Just for readability, I would group slide_resource_downloadable and slide_resource_ids fields definition. That way when reading a model fields, it is easier to understand as you read related fields by group. And because I am a bit maniaque.